### PR TITLE
CMPN-1 - Corrections on endpoints

### PR DESCRIPTION
**Pull request by Icaruspooky**
First structure of the endpoints

### CODE REVIEW

File: src/main/java/com/seals/camplanner/event/controllers/EventController.java [hunk 1]
```diff
@@ -16,9 +17,10 @@
     public List<Event> getEvents() {
         return eventService.findAll();
     }
+
     //TODO add treatment for the "location" parameter
     @PostMapping("/event")
-    public Event saveEvent(@RequestBody Event event) {
+    public Event saveEvent(@RequestBody EventDTO event) {
```
Comment (klimber) [inline src/main/java/com/seals/camplanner/event/controllers/EventController.java:23]: This is a controversial subject, but I agree with [this](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)) source. It states that acronyms should be treated as any other word, ex:
* HTTP Request -> `httpRequest`
* Event DTO -> `eventDto`

What do you think about renaming `EventDTO` to `EventDto`?

File: src/main/java/com/seals/camplanner/event/dto/EventDTO.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.seals.camplanner.event.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
```
Comment (klimber) [inline src/main/java/com/seals/camplanner/event/dto/EventDTO.java:3]: Nitpick: This import is not being used.

File: src/main/java/com/seals/camplanner/event/controllers/EventController.java (hunk 1 above)
Comment (Icaruspooky) [reply src/main/java/com/seals/camplanner/event/controllers/EventController.java:23]: Good one. Thanks.

File: src/main/java/com/seals/camplanner/event/dto/EventDTO.java (hunk 2 above)
Comment (Icaruspooky) [reply src/main/java/com/seals/camplanner/event/dto/EventDTO.java]: Ok. Removed

**Review by klimber — APPROVED** [review]
Ship it! 🚢

**Pull request merged**